magik-debug-adapter/README.md

```diff
@@ -26,6 +28,15 @@ Sourcing ...
 
 Note that the default port the mda uses is `20000`.
 
+On Windows it would look like this:
+
+```batch
+C:\> <path-to-core>\bin\x86\runalias.exe -e /path/to/env -j -agentpath:<path-to-core>\bin\x86\mda.dll=socket,port=32000 swaf
+```
+
+The default port for Windows seems to be `32000` and the .dll does not care what port you specify.
```

Comment:
```suggestion
The default port is `32000`.
```

Perhaps the inability to change the port has changed in recent Smallworld5 versions, but in earlier versions (at least 5.2.8, before that as well I think) the port could definitely be changed.